### PR TITLE
Jarvis

### PR DESCRIPTION
**Pull request by RomainLF**

### CODE REVIEW

**Review by Hamza-Luciano — DISMISSED** [review]
ok for me

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: frontend/src/components/Jarvis/Modal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import SJarvis from "./style";
+
+function Modal({ open, onClose, children }) {
+  if (!open) {
+    return null;
+  }
+
+  return (
+    <SJarvis>
+      <>
```
Comment (LoicBrassart) [inline frontend/src/components/Jarvis/Modal/index.jsx]: You can remove the Fragment, it does nothing in this case

File: frontend/src/components/Jarvis/Modal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <div className="overlay" />
+        <div className="modal">
+          {children}
+          <div className="app">{}</div>
```
Comment (LoicBrassart) [inline frontend/src/components/Jarvis/Modal/index.jsx]: Why `{}` ?
`<div className="app" />` does the same job

File: frontend/src/components/Jarvis/Modal/style.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+export default styled.div`
+  .overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgba(0, 0, 0, 0.7);
+    z-index: 1000;
```
Comment (LoicBrassart) [inline frontend/src/components/Jarvis/Modal/style.jsx]: Meh... That's not top-quality code right here -_-

File: frontend/src/components/Jarvis/Modal/style.jsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  }
+  .modal {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    background-color: #000000;
+    width: 14.4rem;
+    height: 17rem;
+    z-index: 1000;
```
Comment (LoicBrassart) [inline frontend/src/components/Jarvis/Modal/style.jsx]: Same goes here

File: frontend/src/components/Jarvis/index.jsx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  const toggleJarvis = () => {
+    if (active === "jarvisNav") {
+      setActive("jarvisNav jarvisActive ");
+    } else setActive("jarvisNav");
+  };
+  const [isOpen, setIsOpen] = useState(false);
+
+  const search = () => {
+    axios.get(`http://localhost:5000/heroes`).then(({ data }) => {
+      // insert your filter here <3
```
Comment (LoicBrassart) [inline frontend/src/components/Jarvis/index.jsx]: Maybe you could remove my silly comments from your code ? :rofl:

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Some Evils to fix

**Review by Hamza-Luciano — DISMISSED** [review]
ok for us

File: frontend/src/components/Jarvis/Modal/index.jsx [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+  return (
+    <SJarvis>
+              <div className="overlay" />
+        <div className="modal">
+          {children}
+          <div className="app"></div>
+          <div className="border_button">
+            <button onClick={onClose} type="button">
+              Close card
```
Comment (LuigiMenez) [inline frontend/src/components/Jarvis/Modal/index.jsx:17]: a little problem indentation

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
A little problem indentation

**Review by Hamza-Luciano — DISMISSED** [review]
ok for us

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
bad indentation

**Review by Hamza-Luciano — APPROVED** [review]
ok for us

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
No more Evil

**Pull request merged**